api/proto/teleport/legacy/types/types.proto

```diff
@@ -2664,6 +2666,13 @@ message AccessCapabilitiesRequest {
   bool FilterRequestableRolesByResource = 6 [(gogoproto.jsontag) = "filter_requestable_roles_by_resource,omitempty"];
 }
 
+message AccessRequestMode {
+  repeated KubernetesResource KubernetesResources = 1 [
```

Comment:
Can we use a dedicated type for this setting?
Reusing the `KubernetesResource` gives confusion because you can set a lot of data that isn't allowed